src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -383,7 +383,7 @@ FastFreeInProcessHeapFunc __ClrFreeInProcessHeap = (FastFreeInProcessHeapFunc) C
 
 const NoThrow nothrow = { 0 };
 
-#ifdef __llvm__
+#if defined(__llvm__) && !defined(__aarch64__)
```

Comment:
Why is this needed for ARM64? It worries me since that would expose the overriden new / delete to the outside world and it should be internal only.